model.js

```diff
@@ -8,13 +9,54 @@ exports.selectTopics = () => {
 }
 
 exports.selectArticleById = (id) =>{
-    return db.query(`SELECT * FROM articles WHERE article_id = ${id}`).then(({rows}) =>{
+    return db.query(`SELECT * FROM articles WHERE article_id = $1`,[id]).then(({rows}) =>{
         if (rows.length === 0){
             return Promise.reject({
                 status: 404,
-                msg: "Couldn't find article"
+                msg: "Not found"
             })
         }
         return rows[0]
     })
-}
\ No newline at end of file
+}
+
+exports.selectArticles = () =>{
+   const getArticles = () =>{ return db.query(`SELECT * FROM articles ORDER BY created_at DESC`).then(({rows}) =>{
+
+        
+        rows.forEach((article) =>{
+            article.comment_count = 0
+
+            delete article.body
```

Comment:
This could and should be done by selecting the relevant columns using the SQL query so that you don't need to delete the column here.